lib/src/commandlineoptions.cpp

```diff
@@ -144,6 +145,18 @@ CommandLineOptions::ParseResult CommandLineOptions::Parse(int argc, const char*
   string source_filename;           // NOLINT(misc-const-correctness)
   string destination_filename;      // NOLINT(misc-const-correctness)
   string compression_options_text;  // NOLINT(misc-const-correctness)
+  bool overwrite_existing_file{false};
+  bool ignore_duplicate_subblocks{false};
+
+  // specify the string-to-enum-mapping for a boolean option
+  std::map<std::string, bool> map_string_to_boolean{
```

Comment:
Are these matched w/ case-sensitivity?

Reply:
No, they are not. There's a 'case insensitive' passed where this is used.

Reply:
Right, case-insensitive operation is enabled [here](https://github.com/ptahmose/czicompress-zeiss/blob/94f130292bbd28914f469986c04595ab9abc9911/lib/src/commandlineoptions.cpp#L210). And this is what we want here I guess.